tests/xharness/AppRunner.cs

```diff
@@ -132,7 +130,7 @@ async Task<bool> FindSimulatorAsync ()
 		{
 			if (simulators != null)
 				return true;
-			
```

Comment:
This is an unnecessary whitespace change. Please make sure your PR doesn't have any whitespace changes (a quick look shows there are quite a few whitespace changes here). This includes changing between tabs and spaces, changing indentation, adding/removing spaces, adding newlines, etc.

Whitespace changes make pull requests very hard and time-consuming to review, which is why we ask people to make sure their pull requests only have relevant changes.

Reply:
Sorry about that, it was auto-formatted. But.. there are no functional changes in this PR so maybe it's acceptable to have those changes as it is not needed to review the code really?

Reply:
If you want to do whitespace changes, we might accept to do it in a separate PR, with *only* such changes.

Another point to consider is that in case of whitespaces changes it makes the history of the file more complicated (using `git blame` becomes more tedious when half the time when you try to find why a certain line is the way it is you run into someone doing unrelated whitespace changes).

Reply:
Reverted whitespace changes